docs/INSTALL.md

```diff
@@ -18,10 +18,20 @@ mkdir dcos-installer
 cd dcos-installer
 ```
 
+## Initialize Terraform
+
 Run this command below to have Terraform initialized from this repository. There is **no git clone of this repo required** as Terraform performs this for you.
```

Comment:
Use the following command to initialize Terraform from this repository. There is no git clone of this repo required as Terraform performs this for you.

Reply:
Done